common/testing/testlogger/testlogger.go

```diff
@@ -299,16 +299,32 @@ func (tl *TestLogger) PanicOnError(v bool) bool {
 	return tl.state.panicOnError.Swap(v)
 }
 
+func (tl *TestLogger) mergeTags(tags []tag.Tag) []tag.Tag {
```

Comment:
```suggestion
func (tl *TestLogger) getMergedTags(tags []tag.Tag) []tag.Tag {
```
Nit: to me, merge implies to me that you are "merging them in" (modifying the internal tag set) which you are not.